docs/concepts/ipld.md

```diff
@@ -11,4 +11,53 @@ related:
 
 # The InterPlanetary Linked Data (IPLD) model
 
-<ContentStatus />
+The Interplanetary Linked Data (IPLD) model is a single namespace for all hash-inspired protocols. This allows for interoperable data formats. For example, you can reference a Git commit to a Bitcoin transaction.
```

Comment:
While true, I don't think that should be the first paragraph. The idea of linking between different hash-linked system is again mentioned in the third paragraph. So i would remove the first paragraph.